### ISSUE TITLE
Issue with Android 10 or Redmi devices which have android 8 or lower version.

### ISSUE THREAD

**Issue by AnkitBhushan788** [ThanosFisherman/WifiUtils#87]
in Android 10 i have an issue while connecting. Issue is: 

1. It connect WiFi perfectly but when i close my app its automatically disconnect me with the network.
2. In Redmi Devices issue is it ask me to allow WiFi connection continuously and when i allow then still is ask me for permission.

**Comment by ThanosFisherman** [comment]
Yes Xiaomi Redmi devices are quirky with regards to WiFi. I still do not know how to bypass those issues. There is a discussion about possible shortcomings on Redmi devices here #47
